server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -193,6 +203,8 @@ public String typeName() {
     private int ignoreAbove;
     private Integer lengthLimit;
     private String nullValue;
+    @Nullable
+    private Boolean blankPadding;
```

Comment:
Could we default this to `false` and make it non-nullable + primitive? Would reduce the number of possible states.